src/modules/navigation/partials/NavigationTableItem.vue

```diff
@@ -277,6 +277,16 @@ export default {
 </script>
 <style lang="scss">
 
+.app-navigation-entry.active {
+	.icon-share  {
```

Comment:
Ideally we would migrate away from the icon property and instead use the icon slot to provide an inline material icon. That way it can just pick up the text color as with the collapse icon.

Reply:
Uh, icon slot? Not sure what that is :confused:

Reply:
Checkout like 36 for the next entry in the actions here, NcActionButton has two ways of providing an icon, as a css class name or with the icon slot where we can use material inline svg

Reply:
Sorry I thought we can get the other fixes in already :) A follow up PR would still be nice

https://pictogrammers.com/library/mdi/icon/account-plus/ should be there as AccountPlus.vue in the library